hub/powertoys/text-extractor.md

```diff
@@ -31,3 +36,65 @@ From the Settings menu, the following options can be configured:
 | Setting | Description |
 | :--- | :--- |
 | Activation shortcut | The customizable keyboard command to turn on or off always on top for that window. |
+
+## Supported languages
+
+Text Extractor can only recognize languages that have the OCR language pack installed.
+
+The list can be obtained via PowerShell by running the following commands:
+```console
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine, Windows.Foundation, ContentType = WindowsRuntime]
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine]::AvailableRecognizerLanguages
+```
```

Comment:
```suggestion
```powershell
[Windows.Media.Ocr.OcrEngine, Windows.Foundation, ContentType = WindowsRuntime]
[Windows.Media.Ocr.OcrEngine]::AvailableRecognizerLanguages
```
```

Reply:
My change was larger. Having the c: blah blah blocks a copy paste

Reply:
Sorry, missed that part. Removed the PS C:\Windows\system32> prefix as well.